Malay-Kumar-Parida/01.Hello-World/main.go

```diff
@@ -0,0 +1,7 @@
+package main
+
+import "fmt"
+
+func main() {
+	fmt.Println("Hello World")
```

Comment:
```suggestion
	fmt.Println("Hello World")
```
why `fmt.Println`?

Reply:
I used the Println function of the fmt package , which is similar to the stdio in C. I could have used simple Print or Printf function too. But just that Println adds a new line at the end of the output.

Reply:
that's right but why we would require a new line at the end of output here?

Reply:
We actually don't. I just gave it as in C++ we mostly add endl to cout as a practice, so did the same here too. Should I change the function to just Print then?

Reply:
don't mix  `c++` syntax with `golang` initially it happens but try not to. And always it what the program requires not on the language. Here, we don't want to a new line after `Hello World` right? So, don't use `Println` use `Print` instead.

I know these are small things but we have to think about these too. :slightly_smiling_face:

Reply:
Got it Subham, Will keep it in mind. Making the changes too.